src/js/lib/jquery.kazoosdk.js

```diff
@@ -558,6 +558,10 @@
 				'deleteNotification': { verb: 'DELETE', url: 'accounts/{accountId}/notifications/{notificationId}' },
 				'getDnsEntries': { verb: 'GET', url: 'accounts/{accountId}/whitelabel/domains' },
 				'checkDnsEntries': { verb: 'POST', url: 'accounts/{accountId}/whitelabel/domains?domain={domain}' }
+			},
+			onebillIntegration: {
```

Comment:
Method group should match endpoint name for easy "guessing".
```suggestion
			cdrPushOnebill: {
```

Also, please try to keep this map alphabetically sorted, to allow for fast visual lookup.